.fork/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
i get the commands, but this?

Reply:
somehow, gitfork seems to rely on specific formatting, when I let vscode autoformat with our standard prettier settings the new commands disappeared from the context menu... sooo, I added this file to ensure the formatting stays as gitfork seems to require it - and the problem went away